compliance/collection/auditlog/auditlog_impl_test.go

```diff
@@ -54,6 +54,9 @@ func (s *ComplianceAuditLogReaderTestSuite) TestReaderStopDoesNotBlockIfStartFai
 }
 
 func (s *ComplianceAuditLogReaderTestSuite) TestReaderReturnsErrorIfFileExistsButCannotBeRead() {
+	if _, ok := os.LookupEnv("GITHUB_ACTIONS"); ok {
+		s.T().Skip("ROX-14204: This test is not working on GHA.")
+	}
```

Comment:
1. Let's also add a TODO

```suggestion
	// TODO(ROX-14204): enable this test on GHA
	if _, ok := os.LookupEnv("GITHUB_ACTIONS"); ok {
		s.T().Skip("ROX-14204: This test is not working on GHA.")
	}
```

2. Which team is handling https://issues.redhat.com/browse/ROX-14204 ?

Reply:
Ad 2. It's probably on me